content/platforms/kubernetes/kubernetes-installing.md

```diff
@@ -0,0 +1,150 @@
+---
+Title: Installing the Operator
+description: The Redis Enterprise Operator is a software component that must be
+  installed first to facilitate cluster deployments. Installing the operator is
+  a simple process.
+weight: 35
+alwaysopen: false
+categories: ["Platforms"]
+aliases: /rs/concepts/kubernetes/k8s-operator-installation
+---
+
+The Redis Enterprise Operator is a software component that must be
+installed first to facilitate cluster deployments. Some environments provide
+ways to install operators through managed catalogs. For example, the
+[OpenShift OperatorHub](https://docs.openshift.com/container-platform/4.3/operators/olm-adding-operators-to-cluster.html)
+provides an administration interface by which you can find and install the Redis Enterprise
+Operator.
+
+If you are using OpenShift, see [Getting Started with OpenShift and Kubernetes]({{< relref "/platforms/kubernetes/openshift/_index.md" >}}) for
+installation information
+
+If you are using PKS, see [Getting Started with PKS (Pivotal Container Service)]({{< relref "/platforms/kubernetes/pks/_index.md" >}}) for
+installation information.
+
+Otherwise, the operator can be installed on a supported Kubernetes platform
+directly by the following procedure outlined here.
+
+## Overview of the operator
+
+The operator acts on [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRD) that describe Redis Enterprise clusters. The operator itself is a deployment, backed by pods, that runs within your deployment namespace. These operator pods must run with sufficient privileges to create the Redis Enterprise cluster resources within that namespace.
+
+When installed, the following are created:
+
+ * a service account under which the operator will run,
+ * a set of Roles that define the privileges necessary for the operator to perform its tasks,
+ * a set of Role Bindings to the service account to authorize the service account to have the correct roles (see above),
+ * the CRD definition for a cluster,
+ * the operator (a deployment).
+
+ The operator is deployed and runs within single namespace and is scoped to operate only on cluster CRD's in that namespace. This allows a multi-tenant deployment of the operator supporting different versions in different namespaces.
+
+## Getting the operator
+
+The definition and reference materials for the operator are on
+[GitHub](https://github.com/RedisLabs/redis-enterprise-k8s-docs) and the
+implementation is published as a Docker container. The operator
+definitions are packaged as a single generic YAML file called
+[bundle.yaml](https://github.com/RedisLabs/redis-enterprise-k8s-docs/blob/master/bundle.yaml).
+
+The raw bundle can be downloaded via `curl`:
+
+```
+
+curl -O https://raw.githubusercontent.com/RedisLabs/redis-enterprise-k8s-docs/v5.4.14-7/bundle.yaml
```

Comment:
@alexmilowski we shouldn't point to a specific version, otherwise, we'll have to update this doc every 6-8 weeks as we release new builds and versions.

Reply:
We also can't have them pull from the master because that could be a work in progress that is not compatible with the desired version. Maintaining the correct current version might be something we need to handle ... or we change the URL to have {version} in there and give a procedure for finding the version (i.e., looking at the version tags in GitHub or syntax "v" + version)

Reply:
having a placeholder is the better option here I believe. Versioning of docs as a whole would be the preferred solution here. @banker or @bmansheim can comment on when that is planned.

Reply:
I came up with a different solution based on using the GitHub API and releases. You can now pull the latest release with the script or look up a specific release tag yourself.